GrandCentralBoard/Widgets/Bonus/Scene/BubbleScalingController.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  BubbleScalingController.swift
+//  GrandCentralBoard
+//
+//  Created by Bartłomiej Chlebek on 07/04/16.
+//  Copyright © 2016 Oktawian Chojnacki. All rights reserved.
+//
+
+import SpriteKit
+
+protocol BubbleScalingControllerDelegate: class {
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeUp spriteNode: SKSpriteNode)
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeDown spriteNode: SKSpriteNode)
+}
+
+extension BubbleScalingControllerDelegate {
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeUp spriteNode: SKSpriteNode) { }
+    func bubbleScalingController(bubbleScalingController: BubbleScalingController, didScaleSpriteNodeDown spriteNode: SKSpriteNode) { }
+}
+
+final class BubbleScalingController {
```

Comment:
I would change the name of this class. Controller is pretty generic and doesn't mean too much. In our cases we are working with an object responsible for animations - because of that I would change the name to `BubbleScalingAnimator`. That name tells the user of the API that this object is only responsible for scaling animations - nothing more.